pkgs/development/libraries/hunspell/dictionaries.nix

```
@@ -934,4 +934,26 @@ rec {
     readmeFile = "README_pt_PT.txt";
     license = with lib.licenses; [ gpl2 lgpl21 mpl11 ];
   };
+
+  /* PERSIAN */
+
+  fa_IR = fa-ir;
+  fa-ir = mkDict {
+    pname = "hunspell-dict-fa-ir";
+    version = "experimental-2022-09-04";
+    dictFileName = "fa-IR";
+    readmeFile = "README.md";
+    src = fetchzip {
+      url = "https://github.com/nix-julia/lilak-for-nix/releases/download/v3.4-alpha/fa-IR.zip";
+      hash = "sha256-YYCjmUWnY+UoVf7psUYBObitKQkYv/j/JkH0OsATj5I=";
+    };
```

```suggestion
    src = fetchFromGitHub {
      owner = "b00f";
      repo = "lilak";
      rev = "1a80a8e5c9377ac424d29ef20be894e250bc9765";
      hash = "sha256-xonnrclzgFEHdQ9g8ijm0bo9r5a5Y0va52NoJR5d8mo=";
    };
    nativeBuildInputs = [ python3 ];
    buildPhase = ''
      runHook preBuild
      mkdir build
      (cd src && python3 lilak.py)
      mv build/* ./
      runHook postBuild
    '';
```

use the github source rather than a private build. add `python3` to the args at the top of the file and this should work.